fixtures/generated/most-read-geo.ts

```diff
@@ -0,0 +1,296 @@
+/**
+ * DO NOT EDIT THIS FILE!
+ *
+ * This file was automatically generated using the update-manual-fixtures.js script.
+ * Any edits you make here will be lost.
+ *
+ * If the data in these fixtures is not what you expect then
+ *
+ * 1. Update the data in 'fixtures/manual/data', and then
+ * 2. Re-run node 'scripts/test-data/update-manual-fixtures.js'
+ *
+ */
```

Comment:
@oliverlloyd I've updated the header comment so that it correctly identifies the script that is being used to generate this

Reply:
I still feel this is breaking the model here.

the `/manual` folder only contains fixtures that were manaully created. Manual as in, not generated from a script file, that's the distinction.

But here we're explicitly inserting data that is being outpuit from a script and then adding the same script header that gets inserted by the other automatically generated data. You see?

Reply:
The problem here is that I've not updated where this gets moved to. This file is actually in `generated/` rather than `manual/` so I just need to update the header again to reflect this.